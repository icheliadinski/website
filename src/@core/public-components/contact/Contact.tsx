import React from 'react'
import { SideCard } from '@core/ui-kit'

export const Contact = () => {
  return (
    <SideCard>
      <article>Contact me</article>
    </SideCard>
  )
}
