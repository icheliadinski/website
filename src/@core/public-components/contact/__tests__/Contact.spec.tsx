import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import { Contact } from '../Contact'

describe('[Contact]', () => {
  let wrapper: ShallowWrapper

  beforeEach(() => {
    wrapper = shallow(<Contact />)
  })

  it('renders successfully', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
