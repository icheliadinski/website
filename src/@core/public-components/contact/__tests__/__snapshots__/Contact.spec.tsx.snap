// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Contact] renders successfully 1`] = `
<SideCard>
  <article>
    Contact me
  </article>
</SideCard>
`;
